lib/Default/AbstractSmrPlayer.class.php

```diff
@@ -542,6 +544,17 @@ public function isNPC() {
 	public function hasNewbieStatus() {
 		return $this->newbieStatus;
 	}
+	
+	/**
+	 * Does the player have newbie status and
+	 * a low attack rating?
+	 */
+	public function isNoncombatNewbie() {
+		if ($this->getShip()->getAttackRatingWithMaxCDs() <= MAX_ATTACK_RATING_NEWBIE && $this->hasNewbieStatus()) {
```

Comment:
This logic can be simplified with something like
```
return <statement>
```
instead of
```
if (statement)
   return true
else
   return false
```